text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,110 @@
+
+**Note:** In cases where some cores remain unsold in the market, all buyers are obligated to pay the `RESERVE_PRICE`.
+
+#### Renewal Period (7 days)
+
+As the `RENEWAL_PERIOD` commences, all current tenants are granted the opportunity to renew their cores at a slight discount of `MARKET_PRICE * RENEWAL_DISCOUNT` (for instance, 10%). This provision affords marginal benefits to existing tenants, balancing out the non-transferability aspect of renewals.
+
+At the end of the period, all available cores are allocated to the current tenants who have opted for renewal and the participants who placed bids during the market period. If the demand for cores exceeds supply, the cores left unclaimed from renewals may be awarded to bidders who placed their bids early in the auction, thereby subtly incentivizing early participation. If the supply exceeds the demand, all unsold cores are transferred to the Instantanous Market.
+
+#### Reserve Price Adjustment
+
+After all cores are allocated, the `RESERVE_PRICE` is adjusted following the process described in RFC-1 and serves as baseline price in the next `BULK_PERIOD`. 
+
+**Note:** The particular price curve is outside the scope of the proposal. The `MARKET_PRICE` (as a function of `RESERVE_PRICE`), however, is able to capture higher demand very well while being capped downwards. That means, the curve that adjusts the `RESERVE_PRICE` should be more sensitive to undercapacity.
+
+#### Price Predictability 
+
+Tasks that are in the "renewal-pipeline" can determine the upper bound for the price they will pay in any future period. The main driver of any price increase over time is the adjustment of the `RESERVE_PRICE`, that occurs at the end of each `BULK_PERIOD` after determining the capacity fillment of Polkadot UC.  To calculate the maximum price in some future period, a task could assume maximum capacity in all upcoming periods and track the resulting price increase of `RESERVE_PRICE`. In the final period, that price can get a maximum premium of `PRICE_PREMIUM` and after deducting a potential `RENEWAL_DISCOUNT`, the maximum price can be determined.
+
+#### Settlement Period (7 days)
+
+During the settlement period, participants have ample time to trade Coretime on secondary markets before the onset of the next `BULK_PERIOD`. This allows for trading with full Coretime availability. Trading transferrable Coretime naturally continues during each `BULK_PERIOD`, albeit with cores already in use.
+
+
+### Benefits of this system
+- The introduction of a single price, the `RESERVE_PRICE`, provides an anchor for all Coretime markets. This is a preventative measure against the possible divergence and mismatch of prices, which could inadvertently lead to a situation where existing tenants secure cores at significantly below-market rates.
```

Comment:
What we actually should be aiming for is that renewals are more expensive than participating in the open market, at least eventually. Renewals work against allocation efficiency, so they should also be interesting for teams really really needing that consistent 6s coretime, willing to pay a premium to get it.

Reply:
If renewals are sufficiently in sync with the market price (at least eventually), I don't see a reason why we should make it necessary to jump to another core. Jumping to another core, i.e., actively participating in the bidding, has some overhead and I'd prefer the situation where a team just has an account full of DOT that automatically calls renewal in every period and the team just has an eye on the price / balance.

The only good thing about the hopping is that that it is not detrimental if the renewal price runs away in the upper direction, but I don't see why it should generally.

I think it is fine to set the renewal price at the current market price or slightly below.